packages/components/src/menu/styles.ts

```diff
@@ -201,7 +201,7 @@ const baseItem = css`
 			[aria-disabled='true']
 		) {
 		background-color: ${ COLORS.theme.accent };
-		color: ${ COLORS.white };
+		color: ${ COLORS.theme.accentInverted };
```

Comment:
I confirm this is unchanged when comparing to trunk. Hopefully we'll be able to make it themable soonish!

![Screenshot 2024-12-09 at 11 34 36](https://github.com/user-attachments/assets/bb75ddbb-8d2f-404e-9ed9-5893c4d88da0)